### PR TITLE
Correct Survival guide Gil / Tab cost

### PR DESCRIPTION
**Pull request by Xaver-DaRed**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

If anything, it corrects 3 self-contradictory statements

### CODE REVIEW

**Review by TeoTwawki — APPROVED** [review]

**Review by claywar — APPROVED** [review]

**Review by zach2good — APPROVED** [review]

File: scripts/globals/survival_guide.lua
Comment (TeoTwawki) [inline scripts/globals/survival_guide.lua:159]: realized right after hitting the merge button: between cities its 100, if I jump to to a non city like say a bcnm entrance, its 200. This may be where the discrepancy came from.

Comment (Xaver-DaRed) [reply scripts/globals/survival_guide.lua:159]: May be a matter of incorrect base values. I'll check retail, since i dont have the KI

Comment (Xaver-DaRed) [reply scripts/globals/survival_guide.lua:159]: It's not. The base cost is indeed 1000 gil

**Pull request merged**
